packages/SidePanel/src/components/Header/Header.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+
+import { headerCSS } from "./Header.styles";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  hasCloseButton: PropTypes.bool,
+  kind: PropTypes.oneOf(["default", "primary"]),
+  onClose: PropTypes.func,
+};
+
+const defaultProps = {
+  hasCloseButton: true,
+  kind: "default",
+  onClose: () => {},
+};
+
+function darkBackgroundProps(kind) {
+  if (kind === "primary") {
+    return { isDark: true };
+  }
+
+  return {};
+}
+
+const Header = React.forwardRef((props, ref) => {
+  const {
+    hasCloseButton,
+    kind,
+    // injected by Dialog.js
+    onClose,
+    ...moreProps
+  } = props;
+
+  return (
+    <div ref={ref} css={headerCSS} kind={kind} {...moreProps}>
+      <div tabIndex="-1">{props.children}</div>
```

Comment:
I'm not going to suggest we change this now, but I might want to try extending this to be easier to consume.  `children` can remain in a raw container, but if props like `heading`, `onNext`, `onPrev` are provided, I think it would be nice to actually render with a `<Heading>` or `<PrevNextArrows>`.  What do you think?

Reply:
🤔 ohh you mean if I do something like

```
<SidePanel>
  <SidePanel.Header onNext onPrev>some stuff</SidePanel.Header>
</SidePanel> 
```

will render the `<PrevNextArrow />` that's a cool idea! yea totally out of scope for this PR